src/Components/Products/Filters.js

```diff
@@ -0,0 +1,29 @@
+// import './ProductList.css'
+
+import React from 'react'
+import { useProducts } from '../../Context/products-context'
+import FilterData from './FilterData'
+
+const ratingStars = [4,3,2,1]
+
+function Filter() {
+    const {products , setProducts,filterState , dispatchFilterState} = useProducts()
+    const {sortBy , ratingBy , priceRange , showFastDeliveryOnly , showOutOfStock , category} = filterState
+  
+    const {Beds,Lighting,Decor,Sofas} = category
```

Comment:
Accessing these categories seems repetitive. Maybe you could pass these as props to FilterData ?

Reply:
as these categories are not required in Filter.js file , I will remove it from there and only keep in FilterData